src/LaunchDarkly.Client/FeatureFlag.cs

```diff
@@ -110,7 +113,10 @@ private JToken Evaluate(User user, IFeatureStore featureStore, IList<FeatureRequ
                         }
                         catch (EvaluationException e)
                         {
-                            Logger.LogWarning("Error evaluating prerequisites: " + e.Message, e);
```

Comment:
Another example of the exception getting passed as a format argument.  If ```e.Message``` contains {0}, it will get replaced with ```e.ToString()```